src/siembol/templates/job-alerting-deploy-rules.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "siembol.alerting.deploy.rules.fullname" . }}
```

Comment:
this is not needed. storm topologies are deployed by storm topology manager

Reply:
This job sets up ZooKeeper properly.  I don't think this is happening anywhere else, @of13tch?

Reply:
@mariannovotny in my testing with Ollie, the zk nodes needed to be primed with some structure which this job sets up.  I would love that not to be the case, but let's connect and figure this little bit out.

Reply:
@stackedsax these zk nodes are not required to be initialized (except '{}') and will be initialized by sync service with content from git repo. I can explain details how to set up it correctly

Reply:
@of13tch when we looked at this together, I still needed to add more zkCli commands to set zk up correctly.  I guess I could try removing the job and see if everything spins up correctly, but this didn't seem to jibe with our experience.  Do you have any thoughts here?

Reply:
@mariannovotny I tried, I really did!  I removed the job that sets zk up (`job-alerting-deploy-rules.yaml`) but config-editor-rest barfs when it tries to start up.  If siembol is supposed to prime zk without this job, it doesn't seem to be working.